am-lib/src/benchmark/java/uk/gov/hmcts/reform/amlib/AccessManagementServiceBenchmarks.java

```diff
@@ -21,11 +29,16 @@ public void filterResourceBenchmark(BenchmarkState benchmark, DataState data) {
         String accessorId = "user-" + id;
         String accessorRole = "caseworker";
         String resourceId = resourceDefinition.getServiceName() + "-resource-" + id;
+        Map<JsonPointer, SecurityClassification> map = new HashMap<JsonPointer, SecurityClassification>() {
```

Comment:
give map meaningful name

Reply:
Updated the map name to "securityClassification".